oonib/testhelpers/http_helpers.py

```diff
@@ -390,6 +394,9 @@ def tcp_connect(self, socket):
         except ConnectionRefusedError:
             socket_info['status'] = False
             socket_info['failure'] = 'connection_refused_error'
+        except ConnectError:
+            socket_info['status'] = False
+            socket_info['failure'] = 'connect_error'
```

Comment:
Alright, let's open a ticket for this then. I think we should sit sometime in the future and think at how this could be done portably and robustly including also MK into the reasoning.